src/pages/InterviewSchedulingPage/components/InterviewScheduler.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import ScheduleSelector from 'react-schedule-selector';
+
+interface InterviewSchedulerProps {
+  numDays?: number;
+  minTime?: number;
+  maxTime?: number;
+  startDate?: Date;
+  reset?: boolean;
+  setDirtyCallback?: () => void;
+  selectedScheduleCallback?: (arg: Date[]) => void;
+}
+
+function InterviewScheduler({
+  numDays = 5,
+  minTime = 11,
+  maxTime = 21,
+  startDate = new Date(),
+  reset = false,
+  setDirtyCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+  selectedScheduleCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+}: InterviewSchedulerProps): JSX.Element {
+  const [selectedSchedule, setSelectedSchedule] = useState<Date[]>([]);
+
+  const isEqualSchedules = (
```

Comment:
from a performance standpoint i can see why you're comparing schedules on a scheduler by scheduler basis, but I think it's less of a mental burden to do the comparison in the parent.
that way this becomes a stateless purely functional component, with all state in the SchedulersWithConfirmButton component